google/cloud/storage/tests/testbench.py

```diff
@@ -200,6 +200,22 @@ def buckets_get(bucket_name):
     })
 
 
+@gcs.route('/b/<bucket_name>/o')
+def buckets_list(bucket_name):
+    """Implement the 'Objects: list' API: return the objects in a bucket."""
+    base_url = flask.url_for('gcs_index', _external=True)
+    result = {
+        'next_page_token': '',
+        'items': []
+    }
+    for name, o in GCS_OBJECTS.items():
+        if name.find(bucket_name + '/o') != 0:
+            continue
+        print(name)
```

Comment:
Is this print just here for debugging?

Reply:
Ooops, thanks for catching that. Removed.